lib/web/css/source/lib/_forms.less

```diff
@@ -596,7 +596,7 @@
 
     .choice {
         input {
-            vertical-align: top;
```

Comment:
There are general styles for all forms in Magento. Can you please check behavior on the other forms in Magento?